website/docs/reference/options.mdx

```diff
@@ -249,24 +249,24 @@ dynamically with an `http.Handler` or hook into the request chain with an `asset
 
 Not all features of an `http.Request` are currently supported, please see the following feature matrix:
 
-| Feature                 | Win | Mac | Lin |
-| ----------------------- | --- | --- | --- |
-| GET                     | ✅  | ✅  | ✅  |
-| POST                    | ✅  | ✅  | ❌  |
-| PUT                     | ✅  | ✅  | ❌  |
-| PATCH                   | ✅  | ✅  | ❌  |
-| DELETE                  | ✅  | ✅  | ❌  |
-| Request Headers         | ✅  | ✅  | ❌  |
-| Request Body            | ✅  | ✅  | ❌  |
-| Request Body Streaming  | ❌  | ❌  | ❌  |
-| Response StatusCodes    | ✅  | ✅  | ❌  |
-| Response Headers        | ✅  | ✅  | ❌  |
-| Response Body           | ✅  | ✅  | ✅  |
-| Response Body Streaming | ❌  | ❌  | ✅  |
-| WebSockets              | ❌  | ❌  | ❌  |
-
-NOTE: Linux is currently very limited due to targeting a WebKit2GTK Version < 2.36.0. In the future some features will be
-supported by the introduction of WebKit2GTK 2.36.0+ support.
+| Feature                 | Win | Mac | Lin    |
+| ----------------------- | --- | --- | ------ |
+| GET                     | ✅  | ✅  | ✅      |
+| POST                    | ✅  | ✅  | ✅ [^1] |
```

Comment:
@leaanthony , @misitebao does anybody of you know if that footnotes do work with our docs?

Reply:
Ah yeah, always forget about that 😄

Reply:
Oh my bad! I wondered why it looked like nothing had happened. The superscript 1 is super hard to click and it jumps to the bottom of the page. How about a `*` next to is with some text right under it or maybe another column with `Linux (webkit 2.36+)` in it?

Reply:
Sorry already merged 😔, but will open another PR to improve that.